sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/sizes/InputLineLengthExtendedCheck.java

```diff
@@ -1,8 +1,8 @@
-///////////////////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
This change was made 5+ months ago, so I am assuming the CI hasn't been working since then.